assets/js/components/indicators.js

```diff
@@ -288,12 +288,18 @@ export class IndicatorSection {
     this.comparisonButtonsContainer.insertBefore(this.chartContainer);
   }
 
+  highlightComparisonButton(section, muniId) {
+    $(`${section} .chart-btn`).removeClass('active');
+    $(`${section} #${muniId}`).addClass('active');
+  }
+
   _updateComparisonButtons() {
     this.comparisonButtonsContainer.empty();
     this.comparisonButtonsContainer.append('Comparing ');
     [this.chartData(), ...(this.comparisons)].forEach((comparison) => {
-      const button = $(' <button class="button" style="display: unset; margin: 2px 2px"></button> ');
+      const button = $(` <button id="${comparison.municipality.code}"class="button chart-btn"></button> `);
       button.click(() => {
+        this.highlightComparisonButton(this.chart.chart.config.bindto.split(' ')[0], comparison.municipality.code);
```

Comment:
see https://github.com/OpenUpSA/municipal-money-charts/pull/31/files#r1049269018

Reply:
Changed to `this.selector`